docs/docs/build-a-page-with-graphql-query.md

```diff
@@ -2,7 +2,120 @@
 title: Build a page with a GraphQL query
 ---
 
-This is a stub. Help our community expand it.
+Gatsby will create a page for every component located in `src/pages`, with the index (or home) page located at `src/pages/index.js`.
```

Comment:
"Gatsby creates pages from components located within `src/pages`. It watches the folder and creates and removes pages as you add and remove components. The pathname for each page is derived from the name of the component so e.g. `src/pages/about.js` would become `/about/`."